constants.py

```diff
@@ -2,3 +2,4 @@
 
 accepted_channel_id = 539780235411980294
 accepted_message_id = 699757067702894855
+year=2020
```

Comment:
```suggestion
year=2020

```

Can you add a new line here?

Reply:
sure..